examples/group_vars/setup_vars

```diff
@@ -0,0 +1,107 @@
+# TODO: Make hostname a variable instead of the whole API link
```

Comment:
I am unsure whether it makes sense to use as an example `group_vars`, we mostly run on `localhost` all of your roles. I think we should just show how to pass variable file in playbook

Reply:
I will remove the TODO, but I'm not sure that I understood your comment here, do you want me to remove the group_vars?